src/main/drivers/pwm_output.h

```diff
@@ -96,6 +96,13 @@ typedef enum {
     PWM_TYPE_MAX
 } motorPwmProtocolTypes_e;
 
+typedef enum {
```

Comment:
Is this used anywhere outside of `pwm_output.c`? If not, it should better go in there.

Also, n.b. the commonly used format for `typedef` in Betaflight is

```
typedef enum <name>_e {
[...]
} <name>_t;
```

Reply:
I moved the `enum` to `pwm_output.c`. It had to be used in the `dshotCommandControl_t` struct and I overreacted and put the enum in the header.

On the coding style aspect I think you're combining aspects of `enum` and `struct`. For `enum` we always use:
```
typef enum {
    ...
} someName_e;
```
For `struct` we use:
```
typedef struct someName_s {
   ...
} someName_t;
```